galaxy_api/contrib/logging.py

```diff
@@ -0,0 +1,26 @@
+"""Logging utils."""
+
+import boto3
+import watchtower
+import logstash_formatter
+import request_id.logging
+from django.conf import settings
+
+
+class CloudWatchHandler(watchtower.CloudWatchLogHandler):
```

Comment:
Would the watchtower Django specific handler be useful here at all? (https://github.com/kislyuk/watchtower/blob/master/watchtower/django.py)

It seems to have support for setting up the AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION_NAME from django settings. And the log_group / stream_name and other options as kwargs
that could be provided via logging config.

Reply:
With watchtowers  DjangoCloudWatchLogHandler, and providing the other kwargs from settings.py, it should be possible to avoid the need for a custom logging Handler at all.

Reply:
Global settings parameters AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY conflict with `django-storages`. Also since it's highly possible and in fact it is already that CloudWatch and S3 credentials differ, we should avoid using global setting names.